### PR TITLE
add new service getModalName

### PR DESCRIPTION
**Pull request by mgebala8**

### CODE REVIEW

File: src/components/shared/header/index.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import Link from '../../ui/link'
 import SiteNav from './site-nav'
 import UserNav from './user-nav'
 import Hexagons from './hexagons.svg'
+import getModalName from '../../../services/modal.service.js'
```
Comment (Idered) [inline src/components/shared/header/index.js]: This is not how services work. Check to learn about them https://github.com/eyedea-io/zefir#services

File: src/services/modal.service.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export default function getModalName(classNameString){
```
Comment (Idered) [inline src/services/modal.service.js]: This function is specific for optimizely sign up modal. So `getModalName` is to generic for this. Did you even check how other *.service.js files look? Put this function in `src/service/ui.service.js` file and name it `getSignUpModalName`.

Also `className` is enough as argument name.

**Review by Idered — CHANGES_REQUESTED** [review]

File: src/views/landing/index.js [hunk 3]
```diff
@@ -183,7 +182,7 @@ const Landing = ({
 )
 
 Landing.init = ({
-  services: {ui: {toggleModal}}
+  services: {ui: {toggleModal,getModalName}}
```
Comment (mkucharz) [inline src/views/landing/index.js]: Space after comma.

File: src/services/modal.service.js (hunk 2 above)
Comment (Idered) [inline src/services/modal.service.js]: @mgebala8 Change `getModalName` to `getSignUpModalName`.

Comment (Idered) [inline src/services/modal.service.js]: @mgebala8 Change `classNameString` to `className`

File: src/services/ui.service.js [hunk 4]
```diff
@@ -70,4 +70,9 @@ export default class UI {
       }
     })
   }
+
+  getModalName(className){
```
Comment (Idered) [inline src/services/ui.service.js]: Change this to `getSignUpModalName`

Comment (Idered) [reply src/services/ui.service.js]: Or nevermind. I'll fix things myself

**Review by Idered — APPROVED** [review]

**Pull request merged**